src/NSubstitute.Analyzers.Shared/Resources.resx

```diff
@@ -409,4 +409,13 @@
     <value>Non-virtual setup specification.</value>
         <comment>The title of the diagnostic.</comment>
   </data>
+    <data name="SyncOverAsyncThrowsDescription" xml:space="preserve">
+      <value>Sync throws used in async method.</value>
```

Comment:
@dtchepak are you ok with this message?

Reply:
@tpodolak LGTM 👍  Maybe "Synchronous exception thrown from async method" would be a bit clearer but more verbose?

Reply:
Sounds good, changed